Blog post: Getting started with a core initiative

DavidHernandez opened the pull request.

DavidHernandez commented:
Hi @RachelBaram,

Could you do an english review please?

Thanks,
David.

amitaibu commented on app/_posts/2016-08-18-getting-started-with-a-core-initiative.md:
```diff
@@ -0,0 +1,50 @@
+permalink: "/content/getting-started-with-a-core-initiative"
+layout: post
+image: "/assets/images/posts/getting-started-with-a-core-initiative/json-api.png"
+author: DavidHernandez
```
Can you add your data also [here](https://github.com/Gizra/gizra.com/blob/383a4253939cc05b83fff1a53900f9d0019bc69c/_config.yml#L37)

amitaibu commented:
Looks good. Maybe @IshaDakota can have an English review as-well if Rachel is out.

RachelBaram commented:
I can look over it on Sunday.

On Aug 18, 2016 9:56 PM, "Amitai Burstein" <notifications@github.com> wrote:

> Looks good. Maybe @IshaDakota <https://github.com/IshaDakota> can have an
> English review as-well if Rachel is out.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Gizra/gizra.com/pull/478#issuecomment-240821607>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ANpkXyVGXV9jY2ZfQ0ZC6PIVHITX_Uj9ks5qhKragaJpZM4JnVDs>
> .
>

IshaDakota commented:
@DavidHernandez did a [pull request](https://github.com/DavidHernandez/gizra.com/pull/1) on your fork.

DavidHernandez commented:
Merged PR from @IshaDakota.
Added links to D.O user profiles.
Added changes to _config.yml requested by @amitaibu. This last change my add a dependency on this issue: https://github.com/Gizra/gizra.com/issues/430

amitaibu commented:
Closed in favor of #481
